### DISCUSSION TITLE
Stop File view from resetting

### DISCUSSION THREAD

**Discussion by lolbat** [notaapp/talk#329] [Feature Requests] [answered]
I am not sure if this is a feature request or a bug. 

When you scroll the Files section and then drag an image into the editing window the Files section will jump back to the top of the file listing and you click in the editor window. 

I have some very large lists of files that I work with as I add images to the content I write and I often means I have to scroll, drag file, scroll back etc etc

It would be great if the Files view didn't reset

**Comment by astoilkov** [comment, marked as answer]
Yeah. We should improve on that.

I'm also not sure if this is a bug or a feature 😄. Probably a bug.

**Reply by lolbat** [reply]
Awesome. Thanks
